src/components/Pagination/Pagination.js

```diff
@@ -128,21 +127,33 @@ export class RawPagination extends Component {
   }
 
   render() {
-    const pager = new Paginator({
-      currentPage: this.props.currentPage,
-      total: this.props.nbPages,
-      padding: this.props.padding,
-    });
-
-    const createURL = this.props.createURL;
+    const {
+      createURL,
+      pages,
+      isFirstPage,
+      isLastPage,
+      currentPage,
+    } = this.props;
```

Comment:
We could probably refactor the code to prevent destructuring? Since the functions that use those values accepts them as an object. It's kinda obfuscating though. A thought on that @samouss ?